utils/check_dashboard_name_uniqueness.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+const glob = require('glob');
+const {
+  readQuickstartFile,
+  removeRepoPathPrefix,
+  findMainQuickstartConfigFiles,
+} = require('./helpers');
+
+const getQuickstartDashboardConfigs = (quickstartConfigPath) => {
+  const splitConfigPath = quickstartConfigPath.split('/');
+  splitConfigPath.pop();
+  const globPattern = `${splitConfigPath.join('/')}/dashboards/*.+(json)`;
+
+  return glob.sync(globPattern);
+};
+
+const getMatchingNames = (namesAndPaths) => {
+  return namesAndPaths.reduce((acc, { name, path }) => {
+    const duplicates = namesAndPaths.filter(
+      (quickstart) => quickstart.name === name && quickstart.path !== path
+    );
+
+    return [...new Set([...acc, ...duplicates])];
+  }, []);
+};
+
+const cleanQuickstartName = (str) =>
+  str
+    .trim()
+    .toLowerCase()
+    .replace(/\s+/g, '-')
+    .replace(/-+/, '-')
+    .replace(/[^a-z0-9-]/g, '');
+
+const findMatchingDashboardNames = () => {
+  const quickstartMainConfigFilepaths = findMainQuickstartConfigFiles();
+
+  const dashboardConfigs = quickstartMainConfigFilepaths
+    .map((configFile) => {
+      return getQuickstartDashboardConfigs(configFile);
+    })
+    .filter((x) => x.length !== 0)
```

Comment:
[nit] can you rename this to something like `filePaths` instead of `x`

Reply:
Yes :)